### PR TITLE
Skip fragments which inherit legacy modules in debug:fragments

### PR DESCRIPTION
**Pull request by bytehead**
After installation of the `terminal42/notification_center` (just as an example) my `debug:fragments` command is broken:

```
In DebugFragmentsCommand.php line 46:
                                                                              
  Given object is not an instance of the class this property was declared in  
                                                                              

debug:fragments
```

The `LostPasswordController` is registered as a fragment but extends a legacy module (which is fine!):

https://github.com/terminal42/contao-notification_center/blob/ffffe5f78743906de6ef09dcb1ccddda3a6ed190/src/Controller/FrontendModule/LostPasswordController.php#L20

### CODE REVIEW

File: core-bundle/src/Command/DebugFragmentsCommand.php [hunk 1]
```diff
@@ -42,7 +42,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         foreach ($fragments as $identifier => $config) {
             $class = new \ReflectionClass(AbstractFragmentController::class);
-            $attributes = $class->getProperty('options')->getValue($this->container->get($config->getController()));
+
+            try {
```
Comment (aschempp) [inline core-bundle/src/Command/DebugFragmentsCommand.php:46]: should we check for the interface here as well? or something like `hasProperty()`?

Comment (bytehead) [reply core-bundle/src/Command/DebugFragmentsCommand.php:46]: Instead of catching the exception?

**Review by leofeyer — DISMISSED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @bytehead.

**Comment by aschempp** [comment]
Any reason we are hiding the fragments instead of just not rendering attributes? Everything else would still be a valid debug output?

**Pull request merged**